LinaTheExplorer/LinaTheExplorer/1. Best trip hunter/TravelOptions.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Collections;
+
+namespace LinaTheExplorer
+{
+    class TravelOptions : IEnumerable, IEnumerator
```

Comment:
redundant implementation of IEnumerable, IEnumerator. I tried to remove it with the listed methods, the app still works. Removed methods:
- Reset
- Current
- GetEnumerator
- MoveNext